rest-service/manager_rest/config.py

```diff
@@ -19,6 +19,7 @@ class Config(object):
     def __init__(self):
         self._db_address = 'localhost'
```

Comment:
I would actually make localhost be a private case for a host. As the db address should be abstracted by a name, I would change `db_address` and `amqp_address` to a default set of names, and for specific cases, would inject "localhost" instead.